internal/k8s/ako_init.go

```diff
@@ -272,7 +272,7 @@ func (c *AviController) InitController(informers K8sinformers, registeredInforme
 		err = c.FullSyncK8s()
 		if err != nil {
 			// Something bad sync. We need to return and shutdown the API server
-			utils.AviLog.Errorf("Couldn't run full sync successfully on bootup, going to shutdown AKO: %s", err)
+			utils.AviLog.Errorf("Couldn't run full sync successfully on bootup, going to shutdown AKO")
```

Comment:
why did you remove the error?

Reply:
In FullSyncK8s(), we are printing out error and returning it to caller.. So to avoid same error getting printed twice, thought of removing it.